tests-extension/test/ingress_operator.go

```
@@ -0,0 +1,1007 @@
+		defer deleteConfigMap(ctx, kubeClient, "openshift-config", cmName)
+		err = createConfigMap(ctx, kubeClient, "openshift-config", cmName)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By("2. Create a ingresscontroller for OCPBUGS-35027")
+		icName35027 := "bug35027"
+		defer deleteIC(ctx, dynamicClient, icName35027)
+		err = createIC(ctx, dynamicClient, icName35027, icName35027+"."+baseDomain, withPrivateLB(), withClientTLS("client-ca-cert"))
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By("3. Check the custom router pod should not be created for the custom ingresscontroller was abnormal")
+		err = wait.PollUntilContextTimeout(ctx, 5*time.Second, 30*time.Second, true, func(ctx context.Context) (bool, error) {
```

golangci-lint (2.12.2)

[error] ineffectual assignment to err

(ineffassign)